lib/App/DuckPAN/Fathead.pm

```diff
@@ -231,6 +237,26 @@ sub _parse_disambiguations {
 	return \@out;
 }
 
+# Parse list of Related Topics into hash
+# Processed by internal JS to create InfoBox w/ "Related Topics" heading
+sub _parse_related_topics {
+	my ($self, $related_topics, $out) = @_;
+	my @infobox_data;
+	my @related_topics = split /\\n/, $related_topics;
+	foreach my $related_topic (@related_topics){
+		if ($related_topic =~ m/^\[\[(.+)(?:|(.+))?\]\]$/) {
```

Comment:
Uhh, how about `Regexp::Common` for balanced brackets? Would make sense internally too.